include/seastar/core/internal/stall_detector.hh

```diff
@@ -68,14 +68,15 @@ protected:
     cpu_stall_detector_config _config;
     seastar::metrics::metric_groups _metrics;
     friend reactor;
-    virtual bool reap_event_and_check_spuriousness() {
+    virtual bool is_spurious_signal() {
```

Comment:
I renamed this because we no longer "reap" any events here, it is purely a time-based check for spuriousness.